definitions/procedures/packages/enable_version_locking.rb

```diff
@@ -6,7 +6,11 @@ class EnableVersionLocking < ForemanMaintain::Procedure
     end
 
     def run
-      package_manager.install_version_locking(:assumeyes => @assumeyes)
+      installed_fm_packages = []
+      ['satellite-maintain', 'rubygem-foreman_maintain'].each do |pkg|
+        installed_fm_packages << find_package(pkg)
+      end
+      package_manager.reinstall(installed_fm_packages, :assumeyes => @assumeyes)
     end
```

Comment:
I think this whole file can be removed now -- it's unreferenced after you last commit.